client/src/app/components/mission/mission.component.css

```diff
@@ -2,3 +2,26 @@
     display: flex;
     align-items: center;
 }
+
+a {
+  color: coral;
+  font-weight: 600;
+}
+
+.big_picture {
+  height: 90vh;
+  width: 100vw;
+  background-image: url("../../../assets/mission_background_01.jpg");
```

Comment:
../../../assets/ will not work, i need to think what to do about it, i suppose you should hardcode it to https://static.coralreefsource.org/assets/mission_background_01.jpg for now

same for other images in this file